runtime/Cargo.toml

```diff
@@ -5,7 +5,7 @@ homepage = 'https://github.com/galacticcouncil/hydradx-node'
 license = 'Unlicense'
 name = 'hydra-dx-runtime'
 repository = 'https://github.com/galacticcouncil/hydradx-node'
-version = '3.0.0'
+version = '5.0.0'
```

Comment:
3.1.0 rather?

Reply:
I guess by introducing this, old nodes will not be compatible anymore. Since you're changing runtime itself.

Reply:
We have also slightly modified the configuration of the treasury pallet.

Reply:
ok nevermind, it is a breaking change